client/src/components/common/ArtistHeaderSection.tsx

```diff
@@ -95,55 +97,137 @@ const ArtistHeaderSection: React.FC<{ artist: Artist; isManage?: boolean }> = ({
               <HeaderDiv
                 className={css`
-                  align-items: flex-start !important;
+                  align-items: center !important;
+                  padding-bottom: 0 !important;
+                  @media screen and (max-width: ${bp.medium}px) {
+                    margin: 0rem !important;
+                  }
                 `}
               >
-                <div>
-                  <H1>{artist.name}</H1>
-                </div>
-                <div>
-                  {!isManage && <FollowArtist artistId={artist.id} />}
-                  {!isManage && user?.id === artist.userId && (
-                    <Link to={`/manage/artists/${artist.id}`}>
-                      <Button
-                        compact
-                        transparent
-                        type="button"
-                        startIcon={<FaPen />}
-                      >
-                        {t("edit")}
-                      </Button>
-                    </Link>
-                  )}
+                <div
+                  className={css`
+                    ${artistAvatar ? "min-height: 50px;" : ""}
+                    display: flex;
+                    align-items: center;
+                    justify-content: space-between;
+                    width: 100%;
+                  `}
+                >
+                  <div>
+                    <H1
+                      className={css`
+                        @media screen and (max-width: ${bp.medium}px) {
+                          ${!artistAvatar ? "font-size: 1.5rem !important" : ""}
+                        }
+                      `}
+                    >
+                      {artist.name}
+                    </H1>
+                  </div>
+
+                  <div
+                    className={css`
+                      text-align: right;
+                      display: flex;
+                      flex-direction: column;
+                      padding-left: 1rem;
+                      @media screen and (max-width: ${bp.medium}px) {
+                        padding-left: 0.3rem;
+                      }
+                    `}
+                  >
+                    {!isManage && <FollowArtist artistId={artist.id} />}
+                    {!isManage && user?.id === artist.userId && (
+                      <Link to={`/manage/artists/${artist.id}`}>
+                        <Button
+                          compact
+                          transparent
+                          type="button"
+                          startIcon={<FaPen />}
+                        >
+                          {t("edit")}
+                        </Button>
+                      </Link>
+                    )}
+                  </div>
                 </div>
               </HeaderDiv>
-              <MarkdownContent content={artist.bio} />
+              <div
+                className={css`
+                  ${artistAvatar ? "display: none;" : ""}
```

Comment:
The ordering thing here doesn't seem to apply for not just using the ternary operator correctly for both of these